.github/workflows/ci.yml

```
@@ -291,19 +341,35 @@ jobs:
               if [ "$VERSION_TYPE" = "patch" ]; then
                 # Patch versions: Just tag for Sentry tracking (no GitHub release)
                 echo "✅ Production version tagged: v${NEW_VERSION} (patch)"
+              elif [ "$VERSION_TYPE" = "minor" ]; then
+                # Minor versions: Create GitHub Release for new features
+                gh release create "v${NEW_VERSION}" \
+                  --title "Release v${NEW_VERSION} - Minor Update" \
+                  --notes "Minor release v${NEW_VERSION} with new features or enhancements. This release includes backward-compatible functionality additions. PR: ${PR_TITLE}. Sentry tracking: v${NEW_VERSION}, type: Minor. Notes auto-generated by CI/CD." \
+                  --latest
+                
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)